addons/website_sale/static/src/xml/website_sale_utils.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<templates id="template" xml:space="preserve">
+
+	<!-- Products Search Bar autocomplete item -->
+    <div t-name="website_sale.productsSearchBar.autocompleteItem">
+    	<a t-att-href="item.url">
+			<div class="media">
+				<t t-if="products_options.display_image and item.image">
+					<img class="rounded mr-3" t-att-src="'data:image/png;base64,' + item.image"/>
+				</t>
+				<div class="media-body">
+					<h6 t-att-class="item.description ? 'mt-2' : 'mt-3'">
+						<t t-esc="item.name"/>
+						<t t-if="products_options.display_price">
+							<b><span class="float-right" t-esc="item.price" t-options="{'widget': 'monetary', 'display_currency': item.currency_symbol}"/></b>
```

Comment:
I did this to have the currency symbol but I don't understand why it does not work. Do you have an idea ?

Reply:
I don't think there is t-options in static XML... :) That is only for t-field you can use on records.